extensions/wikia/CuratedContent/CuratedContentValidatorController.class.php

```diff
@@ -15,6 +16,11 @@ public function __construct() {
 	}
 
 	public function validateSection() {
+		global $wgRequest;
```

Comment:
above comment makes sense in context of method it applies to. In this particular case I don't see difference